lib/brakeman/processors/erubis_template_processor.rb

```diff
@@ -2,7 +2,7 @@
 
 #Processes ERB templates using Erubis instead of erb.
 class Brakeman::ErubisTemplateProcessor < Brakeman::TemplateProcessor
-  
+
```

Comment:
Unless you are opposed, I'm going to stop undoing the whitespace fixes that happen when I click save in my editor. Maybe over time they will all be fixed 😛. But seriously, if you would prefer me to rip out whitespace changes to preserve the blame log just let me know.